nemesis/menu.py

```diff
@@ -206,15 +206,15 @@ def perform_next_grid(limit: int = 255):
         elif game_vars.nem_checkpoint_ap() == 26:
             nem_gridding_26()
         else:
-            print("----------------------------")
-            print("End of sphere grid, no further grid logic programmed.")
-            print("----------------------------")
+            logger.debug("----------------------------")
+            logger.debug("End of sphere grid, no further grid logic programmed.")
+            logger.debug("----------------------------")
             game_vars.set_nem_checkpoint_ap(
                 game_vars.nem_checkpoint_ap() - 1
             )  # Decrement
         game_vars.set_nem_checkpoint_ap(game_vars.nem_checkpoint_ap() + 1)  # Increment
     # else:
-    # print("###Not enough Slvl:", memory.get_tidus_slvl() - next_ap_needed(game_vars.nem_checkpoint_ap()))
+    # logger.debug("###Not enough Slvl:", memory.get_tidus_slvl() - next_ap_needed(game_vars.nem_checkpoint_ap()))
```

Comment:
Commented out, but this needs to be an f-string or the game will crash if it's un-commented again.

Reply:
I believe this is removed now.